### PR TITLE
Added note about AC add-on requirement to policy packages articles

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7261.

Source:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/6836

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/assign-policy-packages.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ f1keywords:
 
 This article reviews the different ways to assign policy packages to users and groups in Microsoft Teams. Before reading, be sure you've read [Assign policies in Teams - getting started](policy-assignment-overview.md).
 
+> [!NOTE]
+> Each user will require the Advanced Communications add-on in order to recieve a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```
Comment (JohanFreelancer9) [inline Teams/assign-policy-packages.md:28]:
```suggestion
> Each user will require the Advanced Communications add-on in order to receive a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```

File: Teams/manage-policy-packages.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ You can use the [policy packages included in Teams](#policy-packages-included-in
 
 You can customize the settings of the policies in a policy package to suit the needs of your users. When you change the settings of policies in a package, all users who are assigned to that package get the updated settings. You manage policy packages by using the Microsoft Teams admin center or PowerShell.
 
+> [!NOTE]
+> Each user will require the Advanced Communications add-on in order to recieve a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```
Comment (JohanFreelancer9) [inline Teams/manage-policy-packages.md:36]:
```suggestion
> Each user will require the Advanced Communications add-on in order to receive a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```

File: Teams/policy-packages-edu.md [hunk 3]
```diff
@@ -166,6 +166,9 @@ Keep in mind that you can also change the settings of policies in a package afte
 
 Assign the policy package to users. If a user has a policy assigned, and then later you assign a different policy, the most recent assignment will take priority.
 
+> [!NOTE]
+> Each user will require the Advanced Communications add-on in order to recieve a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```
Comment (JohanFreelancer9) [inline Teams/policy-packages-edu.md:170]:
```suggestion
> Each user will require the Advanced Communications add-on in order to receive a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```

File: Teams/policy-packages-gov.md [hunk 4]
```diff
@@ -68,6 +68,9 @@ Keep in mind that you can also change the settings of policies in a package afte
 
 Assign the policy package to users. If a user has a policy assigned, and then later you assign a different policy, the most recent assignment will take priority.
 
+> [!NOTE]
+> Each user will require the Advanced Communications add-on in order to recieve a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```
Comment (JohanFreelancer9) [inline Teams/policy-packages-gov.md:72]:
```suggestion
> Each user will require the Advanced Communications add-on in order to receive a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```

File: Teams/policy-packages-healthcare.md [hunk 5]
```diff
@@ -81,6 +81,9 @@ Keep in mind that you can also change the settings of policies in a package afte
 
 Assign the policy package to users. If a user has a policy assigned, and then later you assign a different policy, the most recent assignment will take priority.
 
+> [!NOTE]
+> Each user will require the Advanced Communications add-on in order to recieve a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```
Comment (JohanFreelancer9) [inline Teams/policy-packages-healthcare.md:85]:
```suggestion
> Each user will require the Advanced Communications add-on in order to receive a custom policy package assignment. For more information, see [Advanced Communications add-on for Microsoft Teams](/microsoftteams/teams-add-on-licensing/advanced-communications).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy reivew.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@KarliStites PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by KarliStites — CHANGES_REQUESTED** [review]
Assign policies to your users in Teams markdown file is assign-policies-users-and-groups.md

**Comment by dariomws** [comment]
@KarliStites thank you for your feedback. I'm only adding the note here.

About assign policies to users, it seems that there are 3 articles with the same content:
policy-assignment-overview.md
assign-policies.md
assign-policies-users-and-groups.md

Actually, the content from policy-assignment-overview.md and assign-policies-users-and-groups.md seems to be merged in assign-policies.md. I think we should embed these articles in assign-policies.md, so instead of updating all of them we can update only one article and the same update will appear in assign-policies.md.

About this PR, do you think I should replace assign-policies.md with assign-policies-users-and-groups.md?

Thank you again!

**Comment by KarliStites** [comment]
@dariomws It depends what information you're trying to reference. 

- assign-policies.md article is an overview article of assigning Teams policies. This article has a table that can be used as a "launching off" point for the different ways to assign policies.
- assign-policies-users-and-groups.md is for individuals and groups, including a batch of users.
- assign-policy-packages.md is specific to policy packages

So, any of those articles will work!

**Comment by KarliStites** [comment]
@dariomws It depends what information you're trying to reference. 

- assign-policies.md article is an overview article of assigning Teams policies. This article has a table that can be used as a "launching off" point for the different ways to assign policies.
- assign-policies-users-and-groups.md is for individuals and groups, including a batch of users.
- assign-policy-packages.md is specific to policy packages

So, any of those articles will work!

**Comment by dariomws** [comment]
Thank you @KarliStites!

It's still not clear to me what updates I should apply to this PR. Do you think I should change/update something?

**Comment by KarliStites** [comment]
@dariomws We have a specific article for assigning policy packages (assign-policy-packages.md) which is a better fit to point to. Currently, you're linking off to assign-policies.md in the Related topics section at the bottom of each of the docs.

**Comment by dariomws** [comment]
@KarliStites done and thank you!

**Pull request merged**